rudder-reports/SOURCES/rudder-reports-postinst

```diff
@@ -1,6 +1,17 @@
 #!/bin/bash
 set -e
 
+# check if a db is configured
+if grep -q rudder.jdbc.url /opt/rudder/etc/rudder-web.properties
+then
+  # if it is an external DB, do not run
+  if grep -qE 'rudder.jdbc.url=jdbc:postgresql://(localhost|127\.0\.)' /opt/rudder/etc/rudder-web.properties
```

Comment:
this detection is invalid, it should be the opposite; with this it skips if it contains localhost